CHANGELOG.md

```diff
@@ -14,6 +14,26 @@ All notable changes to this project will be documented in this file.
 * ### ALL
   * #### Added
   * #### Changed
+    * Repeated capabilities are no longer accessed directly on the session object. Instead there is now a repeated capabilies object for each type of repeated capability that the driver supports. Some drivers do not use repeated capabilities so there is no change for them.
+    * The repeated capabilities object can take integers or strings, a single item or a list.
+    * The repeated capabilities object knows the proper prefix and will add it if needed.
+    * The following are all legal
+        ```python
+        import niscope
+        session = niscope.Session('PXI1Slot2')
+
+        # Channel repeated capabilities
+        session.channel['0'].channel_enabled = True
```

Comment:
since this is a container that you're indexing, shouldn't it be in plural form ("channels")?